### ISSUE TITLE
Breadcrumb component improvements

### ISSUE THREAD

**Issue by rcantin-w** [wellcomecollection/wellcomecollection.org#11209]
This could serve for the upcoming breadcrumb work, but is created with an a11y improvement in mind:

Breadcrumb should be in a `nav` element, and should probably be an ordered list, an their order is of importance.

Feel free to build upon this ticket for new designs and integration work.
